packages/rest/src/__tests__/unit/router/controller-route.unit.ts

```diff
@@ -85,6 +86,59 @@ describe('ControllerRoute', () => {
     expect(route._controllerName).to.eql('my-controller');
   });
 
+  describe('updateBindings()', () => {
+    let appCtx: Context;
+    let requestCtx: Context;
+
+    before(givenContextsAndControllerRoute);
+
+    it('adds bindings to a request context', async () => {
```

Comment:
Since there is always only one request context for every `updateBindings` call, I think `the request context` is more grammatically more correct than `a request context`?

```suggestion
    it('adds bindings to the request context', async () => {
```